.env.local.example

```diff
@@ -3,4 +3,5 @@ NEXT_PUBLIC_PORTIS_APP_ID="PORTIS_APP_ID"
 NEXT_PUBLIC_BN_NOTIFY_API_KEY="BLOCKNATIVE_NOTIFY_API_KEY"
 NEXT_PUBLIC_BN_ONBOARD_API_KEY="BLOCKNATIVE_ONBOARD_API_KEY"
 NEXT_PUBLIC_INFURA_PROJECT_ID="INFURA_PROJECT_ID"
-NEXT_PUBLIC_BLASTAPI_PROJECT_ID="BLASTAPI_PROJECT_ID"
\ No newline at end of file
+NEXT_PUBLIC_BLASTAPI_PROJECT_ID="BLASTAPI_PROJECT_ID"
+NEXT_PUBLIC_COMPETITION_ACTIVE=false
```

Comment:
I made this a new environment variable because that felt like the right level of configurability (e.g. can easily change it without having to deploy new code), but just let me know if anyone thinks this should be handled differently (aka a server-side env var or just hardcoded).